app/assets/javascripts/tip_of_the_days.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Pusty plik do usunięcia. Jest też kilka innych podobnych. 

Generalnie proszę nie używać generowania scaffoldów. Chodzi o to, żeby Państwo nauczyli się samemu tworzyć te pliki.